win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -742,11 +895,15 @@ SetDIBitsToDevice(
         {
             Hit = TRUE;
         }
-        _SEH2_END
+        _SEH2_END;
 
         if (Hit)
         {
-            // We don't die, we continue on with a allocated safe pointer to kernel
+            if (!IS_ALIGNED(Bits, 2)) // If both Read Exception and mis-aligned
+            {
+                LinesCopied = 0;
+                goto Exit;
+            }
            // We don't die, we continue on with a allocated safe pointer to kernel
```

Comment:
unintended control character sneaked in here. Delete that! Also the comment seems to be misplaced.

Reply:
Removed extra control character and relocated comment. Thanks.